packages/backend/src/studio-api-impl.ts

```diff
@@ -149,6 +149,34 @@ export class StudioApiImpl implements StudioAPI {
     return this.environmentManager.getEnvironmentsState();
   }
 
+  async requestRemoveEnvironment(recipeId: string): Promise<void> {
+    const recipe = this.catalogManager.getRecipeById(recipeId);
+    // Do not wait on the promise as the api would probably timeout before the user answer.
+    podmanDesktopApi.window
+      .showWarningMessage(
+        `Delete the environment "${recipe.name}"? This will delete the containers running the application and model.`,
```

Comment:
```suggestion
        `Delete the environment "${recipe.name}" ? This will delete the containers running the application and model.`,
```

Reply:
In English, no space is expected before question/exclam marks